patches/chrome-browser-ui-webui-favicon_source.cc.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/chrome/browser/ui/webui/favicon_source.cc b/chrome/browser/ui/webui/favicon_source.cc
+index 17eca9e53f03a55ecf4f5233b0a4dc85e73ed1b9..05cbc0f56744cdb8d5fc709e5e636a5608626b6e 100644
+--- a/chrome/browser/ui/webui/favicon_source.cc
++++ b/chrome/browser/ui/webui/favicon_source.cc
+@@ -23,9 +23,11 @@
+ #include "components/history/core/browser/top_sites.h"
+ #include "content/public/browser/browser_thread.h"
+ #include "content/public/browser/web_contents.h"
++#if !defined(OS_ANDROID)
+ #include "extensions/browser/extension_registry.h"
+ #include "extensions/common/constants.h"
+ #include "extensions/common/manifest.h"
++#endif
+ #include "net/url_request/url_request.h"
+ #include "third_party/skia/include/core/SkBitmap.h"
+ #include "ui/base/layout.h"
+@@ -109,6 +111,7 @@ void FaviconSource::StartDataRequest(
+     return;
+   }
+ 
++#if !defined(OS_ANDROID)
```

Comment:
this is an upstream issue, but it should have `#if BUILDFLAG(ENABLE_EXTENSIONS)` not `#if !defined(OS_ANDROID)`
and `#include "extensions/buildflags/buildflags.h"` should be added in a chromium_src override

Reply:
looks like you made changes here https://github.com/brave/brave-core/pull/5102/commits/04be0228be7ca4e38fa261bb62b3a18bc726d98f

Reply:
cc: @samartnik